lib/derailed_benchmarks/tasks.rb

```diff
@@ -1,8 +1,14 @@
 require_relative 'load_tasks'
 
 namespace :perf do
+  desc "runs the performance test against two most recent commits of the current app"
+  task :app do
+    ENV["DERAILED_PATH_TO_LIBRARY"] = '.'
+    Rake::Task[:library].invoke
+  end
+
   desc "runs the same test against two different branches for statistical comparison"
-  task :library do
+  task :library => [:app] do
```

Comment:
You don't need to do this, the loading of the app is done when the sub script gets executed